templates/base.html

```diff
@@ -131,6 +131,8 @@
 mixpanel.init("{{ mixpanel_key }}", { track_pageview : false, debug: {{ is_debug }} });</script><!-- end Mixpanel -->
 {% endif %}
 
+  <link rel="search" type="application/opensearchdescription+xml" title="Quay" href="/opensearch.xml" />
```

Comment:
The title needs to come from the config: `REGISTRY_TITLE` or `REGISTRY_TITLE_SHORT`. Both should already be in the template's context

Reply:
Done